lib/improver/tests/utilities/test_cube_manipulation.py

```diff
@@ -1297,5 +1301,154 @@ def test_multiple_cell_methods_in_input_cube(self):
                          (extra_cell_method, expected_cell_method,))
 
 
+class Test_sort_coord_in_cube(IrisTest):
+    """Class to test the sort_coord_in_cube function."""
+
+    def setUp(self):
+        """Set up a cube."""
+        self.ascending_height_points = np.array([5., 10., 20.])
+        cube = set_up_height_cube(self.ascending_height_points)[:, 0, :, :, :]
+        data = np.ones(cube.shape)
+        data[1] = np.zeros(cube[1].shape) + 2
+        data[2] = np.zeros(cube[2].shape) + 3
```

Comment:
See comments in test_mathematical_operations.

Reply:
Changed.